README.md

```diff
@@ -114,7 +114,7 @@ These are key to many of the shortcuts utilized by tiling window managers. This
 
 > This mode is activated with `Super` + `Return`.
 
-Window management mode activates additional keyboard control over the size and location of the currently-focused window. The behavior of this mode changes slightly based on whether you are in auto-tile mode, or in the default stacking mode. In the default mode, an overlay is displayed snapped to a grid, which represents a possible future location and size of your focused window. This behavior changes slightly in auto-tiling mode, where resizes are performed immediately, and overlays are only shown when swapping windows.
+Window management mode activates additional keyboard control over the size and location of the currently focused window. The behavior of this mode changes slightly based on whether you are in the auto-tile mode or the default stacking mode. In the default mode, an overlay is displayed snapped to a grid, which represents a possible future location and size of your focused window. This behavior changes slightly in the auto-tiling mode where resizes are performed immediately, and overlays are only shown when swapping windows.
```

Comment:
The hyphen between `currently` and `focused` is not grammatically necessary (`ly` already conveys that the next word is also a modifier), so that change is fine.

`the auto-tile mode` doesn't sound quite right to me. `auto-tile mode` is the name of a specific mode, so even though it's not a proper noun, it may not need an article, similar to the non-proper names of specific sports, academic subjects, etc. The reason `the` is used in `the default stacking mode` is because `default` is a definite noun but isn't a specific name; it wouldn't be needed if the word `default` wasn't there. (By the way, we might want to rename "stacking mode" at some point since "stacking" is now a different feature within tiling mode.)

The comma removal in the last sentence is incorrect. We're not referring to `the auto-tiling mode where resizes are performed immediately` as opposed to some other auto-tiling mode where that's not the case; `This behavior changes slightly in auto-tiling mode` is an independent clause, and the rest of the sentence is a dependent clause that elaborates on the change. If you want to remove a comma here, the second one (between `immediately` and `and`) _is_ grammatically incorrect and should be removed (while `overlays are only shown when swapping windows` might sound like an independent clause, the `where` applies to it as well since it's talking about auto-tiling mode.)